docs/getting_started.rst

```diff
@@ -170,5 +210,5 @@ between ``PauliOp`` objects and ``SparsePauliOp`` objects from Qiskit:
     P = fp.PauliString('XYZ')
     qiskit_pauli = fp.to_qiskit(P)
```

Comment:
in Qiskit Integration section we should replace `Fast-Pauli` entries with  `fast_pauli` to be more consistent